app/controllers/email_addresses_controller.rb

```diff
@@ -0,0 +1,74 @@
+class EmailAddressesController < ApplicationController
+  before_action :set_email_address, only: [:show, :edit, :update, :destroy]
+
+  # GET /email_addresses
+  # GET /email_addresses.json
+  def index
+    @email_addresses = EmailAddress.all
+  end
+
+  # GET /email_addresses/1
+  # GET /email_addresses/1.json
+  def show
+  end
+
+  # GET /email_addresses/new
+  def new
+    @email_address = EmailAddress.new(person_id: params[:person_id])
+  end
+
+  # GET /email_addresses/1/edit
+  def edit
+  end
+
+  # POST /email_addresses
+  # POST /email_addresses.json
+  def create
+    @email_address = EmailAddress.new(email_address_params)
+
+    respond_to do |format|
+      if @email_address.save
+        format.html { redirect_to @email_address.person, notice: 'Email address was successfully created.' }
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.